.node-version

```diff
@@ -0,0 +1 @@
+14.18.2
```

Comment:
Interesting, yeah... we do for the integration tests run [here](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/.github/workflows/integration-test.yml#L84). I honestly just added it as a convenience so that it stands by itself, but to be honest I'm not sure if we ever build this project without being "nested" inside of the Dashboards codebase.